src/app.module.ts

```diff
@@ -1,9 +1,28 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { User } from './user/entities/user.entity';
+import { UserModule } from './user/user.module';
+import { AuthModule } from './auth/auth.module';
 import { AppService } from './app.service';
-
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('dotEnv').config();
```

Comment:
(suggestion; blocking): Nest offers a library for loading environment variables that we can use:

```TypeScript
import { Module } from '@nestjs/common';
import { ConfigModule } from '@nestjs/config';

@Module({
  imports: [ConfigModule.forRoot()],
})
export class AppModule {}